syntaxes/v.tmLanguage.json

```diff
@@ -267,7 +267,7 @@
 							"name": "meta.expr.numeric.cast.v"
 						},
 						{
-							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune)(?=\\s*\\()",
+							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune|size_t)(?=\\s*\\()",
```

Comment:
Can you remove ustring please? It was removed recently.

Reply:
Yeah sure.

Reply:
Thanks :)